### ISSUE TITLE
Problema com uma função

### ISSUE THREAD

**Issue by arthurscheidt** [fsouza99/discord-message-deleter#1]
Unexpected error: Message: no such element: Unable to locate element: {"method":"css selector","selector":"[aria-label="Limpar busca"]"}

teria uma versão atualizada ou será se to errando em algo?

**Comment by fsouza99** [comment]
O código depende do DOM da aplicação, então uma mudança por lá pode gerar esse tipo de erro, e foi isso que ocorreu aqui.
Obrigado pelo comentário.

**Comment by arthurscheidt** [comment]
valeu!!! e outra coisa se quiser deixar o readme mais acessível é que precisa por em o discord em ptbr
